src/main/java/com/microsoft/jenkins/artifactmanager/AzureArtifactConfig.java

```diff
@@ -106,7 +106,7 @@ public ListBoxModel doFillStorageCredentialIdItems(@AncestorInPath Item owner) {
         }
 
         public FormValidation doCheckContainer(@QueryParameter String container) {
-            boolean isValid = Utils.validateContainerName(container);
+            boolean isValid = Utils.containTokens(container) || Utils.validateContainerName(container);
```

Comment:
code looks good, any chance of a test though?

Should be able to just do a plain unit test, don't need a Jenkins integration test

Reply:
done